FpsCheck.py

```diff
@@ -243,7 +243,12 @@ def is_eTLD_Plus1(self, site):
         Returns:
             boolean with truth value dependent on value of get_public_suffix
         """
-        return self.etlds.get_public_suffix(site, strict=True) is not None
+        assert site is not None
+        site = site.replace("https://", "", 1)
+        has_etld = self.etlds.get_sld(site, strict=True) == site
+        is_more_than_etld = self.etlds.get_tld(site, strict=True) != site
+        return has_etld and is_more_than_etld
```

Comment:
My thought was thought calling the first variable `is_etldp1` is also a bit confusing since an entry that is itself an effective top level domain would return true for that. I think it might be confusing to mark that an etld is an etld+1, but I'm not sure if there's a better option than what you're suggesting.

Reply:
I'm not entirely clear on the semantics of the publicsuffix2 API, but I was guessing that the use of `strict=True` opted out of that behavior and would make the API return `None` if we gave it an eTLD. Can you test it out to be sure?

Alternatively, I think it'd be fine to do something like:

```
is_etldp1_or_etld = self.etlds.get_sld(site, strict=True) == site
is_etld = self.etlds.get_tld(site, strict=True) == site
return is_etldp1_or_etld and not is_etld
```

Since that makes it clearer what the APIs return, and what exactly we're looking for, IMO.

Reply:
I have tested it out, and it doesn't work as intended with slds without the inclusion of the `get_tld` call, as I was referencing [here](https://github.com/GoogleChrome/first-party-sets/pull/53#discussion_r1274114673). 

> I'm not entirely clear on the semantics of the publicsuffix2 API, but I was guessing that the use of `strict=True` opted out of that behavior and would make the API return `None` if we gave it an eTLD. Can you test it out to be sure?

That's what one would think, but the inclusion of `strict=True` is what allows the function to actually refer to the PSL at all. Without the argument, the methods in publicsuffix2 don't do much of anything - they just extract the sld/tld-like section of a url-like string. Here's the relevant part of the description of `get_sld`:

> The logic does not check by default whether the TLD is in the Trie, so for example, 'www.this.local' will return 'this.local'. If you want to ensure the TLD is in the public suffix list, use strict=True.

It's pretty silly IMO.

Reply:
Got it, yeah you did mention that there's some overlap between the APIs. +1 that it's not the easiest to use, but I think we're using it correctly now at least.